lib/rex/parser/nexpose_raw_nokogiri.rb

```diff
@@ -193,6 +193,13 @@ def report_vuln(&block)
       vuln_instances = @report_data[:vuln][:matches].size
       db.emit(:vuln, [refs.last,vuln_instances], &block) if block
 
+      # TODO: potential remove the size limit on this field, might require
+      # some additional UX
+      if @report_data[:vuln]['title'].length > 255
```

Comment:
It can be as long as it is a `varchar`, but if we change the column to `text` we will get `nil` back.